src/pages/Login/index.tsx

```diff
@@ -54,7 +56,7 @@ export default function Login() : JSX.Element {
 
   const signIn = async ({ email, password }: LoginForm): Promise<void> => {
     try {
-      const response = await auth().signInWithEmailAndPassword(email, password);
+      const response = await userAPI.signInWithEmailAndPassword(email, password);
```

Comment:
Just rename to `Users.signInWithEmailAndPassword`, or simply `Users.signIn`

Reply:
Fixed by new commit.